sale_stock_delivery_address/__manifest__.py

```diff
@@ -1,14 +1,14 @@
 # Copyright 2020-21 ForgeFlow S.L.
-# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 
 {
     "name": "Sale Stock Delivery Address",
     "version": "14.0.1.0.1",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/sale-workflow",
     "category": "Sales Management",
-    "license": "LGPL-3",
-    "development_status": "Production/Stable",
+    "license": "AGPL-3",
```

Comment:
Before modifying this, I would have asked author

Reply:
This is not something that the author has to decide: they are not complying with the licenses. They can later think about relicensing whole chain. For now, I'm only fixing what is not correct.

Reply:
I didn't see licensing error in tests

Reply:
https://github.com/OCA/sale-workflow/runs/6948099807?check_suite_focus=true#step:5:5

Reply:
@LoisRForgeFlow be aware of this license change for complying with dependency chain to proceed with the actions you prefer.